packages/boxed-expression-component/src/contextMenu/MenuWithHelp/MenuItemWithHelp.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import "./MenuWithHelp.css";
+import { MenuItem, MenuItemAction } from "@patternfly/react-core/dist/js/components/Menu";
+import { HelpIcon } from "@patternfly/react-icons/dist/js/icons/help-icon";
+
+export interface MenuItemWithHelpProps {
+  menuItemKey: string;
+  menuItemCustomText?: string;
+  menuItemDescription: string;
+  menuItemIcon?: any;
+  menuItemIconStyle?: any;
+  setVisibleDescription: (description: string) => void;
+  getVisibleDescription: () => string;
+}
+
+export function MenuItemWithHelp({
```

Comment:
So..this is called `MenuItemWithHelp`, but all the props refer to `description`. I think that's confusing, we should either stick with `description` or `help`. Either are fine for be, as long as everything references the same thing by a single name -- `help` or `description`.

Reply:
done as abcc13f78d